compiler/optimizer/VPHandlers.cpp

```diff
@@ -1540,7 +1547,8 @@ TR::Node *constrainAload(OMR::ValuePropagation *vp, TR::Node *node)
          vp->addGlobalConstraint(node, TR::VPObjectLocation::create(vp, TR::VPObjectLocation::J9ClassObject));
          }
 
-      addKnownObjectConstraints(vp, node);
+      if (addKnownObjectConstraint(vp, node))
```

Comment:
Should this be a call to `addKnownObjectConstraints()` (note the name is plural)?

Reply:
My bad, I forgot that I have made this little change after tested my branch. I've opened https://github.com/eclipse/omr/pull/1978 for it. @vijaysun-omr Can you review and merge it? I have tested it this time, and so sorry for the trouble.